keyboards/crkbd/keymaps/rustyjonez/keymap.c

```diff
@@ -0,0 +1,343 @@
+      KC_LSFT, KC_A, KC_S, KC_D, KC_F, KC_G,                           KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT,\
+      KC_LCTL, KC_Z, KC_X, KC_C, KC_V, KC_B,                                  KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_SFTENT,\
+                            KC_LALT, MO(_NUM), LT(_NAV,KC_SPC),               LT(_NUM,KC_SPC), MO(_SYMBOL), LT(_FUNCT,KC_ESC)\
+    ),
+
+  [_NUM] = LAYOUT( \
+      KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5,                                   KC_6, KC_7, KC_8, KC_9, KC_0, KC_DEL,\
+      KC_LSFT, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5,                             KC_PPLS, KC_4, KC_5, KC_6, KC_MINS, KC_EQL,\
+      KC_LCTL, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12,                          KC_PSLS, KC_1, KC_2, KC_3, KC_SLSH, KC_RBRC,\
+                                    KC_LGUI, KC_TRNS, KC_F5,                  KC_TRNS, KC_0, KC_PDOT\
+    ),
+
+  [_NAV] = LAYOUT( \
+      KC_TAB, KC_EXLM, KC_HOME, KC_UP, KC_END, KC_PGUP,                       KC_NO, KC_HOME, KC_UP, KC_END, KC_PGUP, KC_BSPC,\
+      KC_TRNS, LCTL(KC_A), KC_LEFT, KC_DOWN, KC_RGHT, KC_PGDN,                KC_NO, KC_LEFT, KC_DOWN, KC_RGHT, KC_PGDN, KC_DEL,\
+      KC_TRNS, LCTL(KC_Z), LCTL(KC_X), LCTL_T(KC_C), LCTL(KC_V), LCTL(KC_B),  KC_ESC, KC_NO, KC_NO, KC_NO, LCTL(KC_MINS), LCTL(LSFT(KC_MINS)),\
+                                    KC_TRNS, KC_NO, KC_TRNS,                  KC_F5, KC_TRNS, LCTL(KC_F5)\
+    ),
+
+  [_FUNCT] = LAYOUT( \
+      RGB_TOG, KC_NO, KC_NO, KC_NO, KC_NO, KC_CALC,                           KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6,\
+      RGB_MOD, RGB_HUI, RGB_SAI, RGB_VAI, RGB_SPI, KC_MYCM,                   KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12,\
+      RGB_RMOD, RGB_HUD, RGB_SAD, RGB_VAD, RGB_SPD, KC_NO,                    RGB_MOD, RGB_HUI, RGB_SAI, RGB_VAI, RGB_VAD, TG(_GAME),\
+                                    KC_VOLD, KC_VOLU, KC_MUTE,                RGB_TOG, RGB_SAD, KC_TRNS\
+    ),
+
+  [_GAME] = LAYOUT( \
+     KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T,                                    KC_Y, KC_U, KC_I, KC_O, KC_P, KC_BSPC,\
+     KC_TAB, KC_A, KC_S, KC_D, KC_F, KC_G,                                    KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT,\
+     KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B,                                   KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_ENT,\
+                                    KC_ESC, KC_LCTL, KC_SPC,                  KC_SPC, KC_NO, TG(_GAME)\
+    ),
+
+  [_SYMBOL] = LAYOUT( \
+     KC_TILD, KC_EXLM, KC_AT, KC_HASH, KC_DLR, KC_PERC,                        KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_NO,\
+     KC_LSFT, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN,                    KC_NO, KC_DLR, KC_PERC, KC_CIRC, KC_UNDS, KC_PLUS,\
+     KC_LCTL, KC_PSLS, KC_PIPE, KC_BSLS, KC_LBRC, KC_RBRC,                      KC_NO, KC_EXLM, KC_AT, KC_HASH, KC_QUES, KC_NO,\
+                                    KC_NO, KC_LCBR, KC_RCBR,                      KC_NO, KC_TRNS, KC_NO\
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

Reply:
Glad to know these are unnecessary, I found them ugly too! Thanks!